exercises/series/series.exs

```diff
@@ -0,0 +1,11 @@
+defmodule StringSeries do
+  @doc """
+  Given a string `s` and a positive integer `size`, return all substrings
+  of that size. If `size` is greater than the length of `s`, or less than 1,
+  return an empty list.
+  """
+  @spec slices(s :: String.t(), size :: integer) :: any
```

Comment:
Good catch. Missed that when I copied this over from my earlier work on robot-simulator.